.github/workflows/ci.yaml

```diff
@@ -0,0 +1,34 @@
+name: CI
+on: [push, pull_request, workflow_dispatch]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repo
+      uses: actions/checkout@v2
+
```

Comment:
Why empty lines here?

Reply:
just readability, I can remove if you really don't like those lines, but I  think it's easier to read blocks like this.

Reply:
Curious if this is still true if we drop names

Reply:
I'm a fan of the explicit names, is there any reason you don't want them?  It makes the file a lot clearer if each step is named appropriately and the run summary becomes clearer too.

Reply:
IMO it's just unnecessary aliasing. The names (as they were), were already quite explicit to me. If they weren't explicit to you, that's probably a good signal that naming them could reduce confusion :). I prefer the more brief form since it's already reasonably named, but it's not a hill that I'm willing to die on and I think your perspective on this is at least (if not more) valid than mine.